src/mappings/geyser.ts

```diff
@@ -12,7 +12,7 @@ import {
   GysrSpent
 } from '../../generated/templates/Geyser/Geyser'
```

Comment:
we're also going to need to add an event handler for `OwnershipTransferred` and update the `Geyser.owner` field accordingly

Reply:
Nice catch, I'll include that.